basics.tex

```diff
@@ -466,7 +466,7 @@ \section{Types are higher groupoids}
     \begin{equation*}
       \prd{z,w:A}{q:x= z}{r:z= w} \big(\refl{x}\ct(q\ct r)= (\refl{x}\ct q)\ct r\big).
     \end{equation*}
-    To construct an element of this type, let $D_2:\prd{x,z:A}{q:x=z} \type$ be the type family
+    To construct an element of this type, let $D_2:\prd{x,z:A} (x=z) \to \type$ be the type family
```

Comment:
I prefer (x=z); otherwise my eye takes a moment to reject the parsing x=(z\to\type).